007_multiply_array_elements_with_constant.js

```diff
@@ -0,0 +1,15 @@
+const readline = require("readline-sync");
+
+const multiplyArrayWithConstant = (elements) => {
```

Comment:
pass constant as an argument to the function.

Reply:
Ok.